assets/src/components/Sidebar/CampaignSidebar.js

```diff
@@ -31,6 +31,31 @@ const loadOptions = async (value) => {
   const response = await fetch(optionsJsonUrl);
   return await response.json();
 }
+const makeDefaultOrNull = ( result, field ) => {
+  // Adding this check to prevent a crash. Probably the previous code can be rewritten to not produce null, but
```

Comment:
This comment was already on the code but it's still accurate. The code will be rewritten in the next PR.